packages/runtime/runtime-definitions/src/garbageCollection.ts

```diff
@@ -16,6 +19,8 @@ export interface IGCData {
  * Represents the format of the GC details that is stored in the summary for each node.
  */
 export interface IGCDetails {
+    /** Whether this node is in use or not. */
+    used: boolean;
```

Comment:
referenced ? Used does not tell me much (when I look at code where we set this property)

Reply:
I was debating between referenced and used. referenced sounds good too.